mkdocs/docs/HPC/account.md

```diff
@@ -208,7 +208,7 @@ OpenSSH_7.4p1, OpenSSL 1.0.2k-fips 26 Jan 2017
 To access the clusters and transfer your files, you will use the
 following commands:
 
-1.  ssh-keygen: to generate the ssh keysgenerate a public
+1.  ssh-keygen: to generate the ssh keys;
```

Comment:
While you're at it, make this:

```suggestion
1.  `ssh-keygen`: to generate the SSH key pair (public + private key);
```

Also use backticks below for command names